src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-alerts-zwave_js.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  PropertyValues,
+  TemplateResult,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { DeviceRegistryEntry } from "../../../../../../data/device_registry";
+import {
+  ConfigEntry,
+  getConfigEntries,
+} from "../../../../../../data/config_entries";
+import {
+  getZwaveJsIdentifiersFromDevice,
+  ZWaveJSNodeIdentifiers,
+  ZwaveJSNodeComments,
+  fetchZwaveNodeComments,
+} from "../../../../../../data/zwave_js";
+import { haStyle } from "../../../../../../resources/styles";
+import { HomeAssistant } from "../../../../../../types";
+
+@customElement("ha-device-alerts-zwave_js")
+export class HaDeviceAlertsZWaveJS extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public device!: DeviceRegistryEntry;
+
+  @state() private _entryId?: string;
+
+  @state() private _configEntry?: ConfigEntry;
+
+  @state() private _multipleConfigEntries = false;
+
+  @state() private _nodeId?: number;
+
+  @state() private _nodeComments?: ZwaveJSNodeComments;
+
+  protected updated(changedProperties: PropertyValues) {
+    if (changedProperties.has("device")) {
+      const identifiers: ZWaveJSNodeIdentifiers | undefined =
+        getZwaveJsIdentifiersFromDevice(this.device);
+      if (!identifiers) {
+        return;
+      }
+      this._nodeId = identifiers.node_id;
+      this._entryId = this.device.config_entries[0];
+
+      this._fetchNodeDetails();
+    }
+  }
+
+  protected async _fetchNodeDetails() {
```

Comment:
```suggestion
  private async _fetchNodeDetails() {
```
Also should be moved below render